docs/framework/release-notes/2024/04-april-security-and-quality-rollup

```diff
@@ -0,0 +1,101 @@
+# April 2024 Security and Quality Rollup
+
+_Released April 9, 2024_
+
+## Summary of What's New in this Release
+
+### Security improvements
+
+#### CVE-2024-21409 – Remote Code Execution Vulnerability
+
+This security update addresses a remote code execution vulnerability detailed in [CVE 2024-21409](https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE 2024-21409).
+
+#### Defense in Depth Vulnerability
+
+This security update addresses an issue where version of the OSS zlib library is out of date.
+
+#### Defense in Depth Vulnerability
+
+This security update addresses an issue in AIA fetching process.
+
+### Quality and Reliability Improvements
+
+This release contains the following quality and reliability improvements.
+
+####ASP.NET
++ Addresses an issue with JavaScriptSerializer where after installing the January Security and Quality update there is a performance degradation. (*applies to: .NET Framework 4.6.2, 4.7, 4.7.1,4.7.2, 4.8, 4.8.1*)
+
+####CLR
++ Addressed an issue with thread pool recycled lists become unresponsive on multi-CPU-group computers (*applies to: .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8, 4.8.1*)
++ Addressed an issue where Interlocked.Read from 32-bit apps are much slower on some computers. (*applies to: .NET Framework 4.8.1*)
+
+####Compilers
++ Addresses an issue using the native C# compiler (csc.exe) to compile code making calls to COM interop assemblies. (*applies to: .NET Framework 4.8.1*)
+
+## Known Issues in this Release
+
+This release contains no known issues.  
+
+
+## The following tables outline the updates in this release. 
+
+| Product Version | Cumulative Update |
+| --- | --- |
+| **Microsoft server operating system, version 23H2** | |
+| .NET Framework 3.5, 4.8.1 | [5036617](https://support.microsoft.com/kb/5036617) |
+| **Windows 11, version 22H2 and Windows 11, version 23H2** | |
+| .NET Framework 3.5, 4.8.1 | [5036620](https://support.microsoft.com/kb/5036620) |
+| **Windows 11, version 21H2** | **[5037037](https://support.microsoft.com/kb/5037037)** |
+| .NET Framework 3.5, 4.8 | [5036611](https://support.microsoft.com/kb/5036611) |
+| .NET Framework 3.5, 4.8.1 | [5036619](https://support.microsoft.com/kb/5036619) |
+| **Microsoft server operating system, version 22H2 and version 21H2** | **[5037033](https://support.microsoft.com/kb/5037033)** |
```

Comment:
will this be for both version 22h2 and 21h2?

Reply:
fixed